frontends/smv_node.h

```diff
@@ -19,20 +18,46 @@ struct SMVnode
     TermNode,
     SortNode
   };
+  enum Type
+  {
+    Signed,
+    Unsigned,
+    Integer,
+    Real,
+    Boolean,
+    BV,
+    BVnot
```

Comment:
I remember you mentioning this, but could you explain it again for me please? So the BV tag is for an edge case in Boolector right? Because otherwise I would expect it to just be Signed/Unsigned. And then, what is BVnot for? Is that like the bit-vector operator bitwise NOT?